lib/components/Input/base/index.tsx

```diff
@@ -186,6 +209,21 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
                                     )}
                                 </label>
                             )}
+                            {rightPlaceholder &&
+                                (!label || (label && (value || focused))) &&
+                                (
+                                    <Text
+                                        size={inputSize}
+                                        as="span"
+                                        className={clsx(
+                                            "quill-input-label__label",
+                                            "quill-input-label__label--right",
+                                            `quill-input-label__label--disabled`,
+                                        )}
+                                    >
+                                        {rightPlaceholder}
+                                    </Text>
+                                )}
```

Comment:
it looks like leftPlaceholder and rightPlaceholder have exactly the same logic, can we move it out and reuse for both? It can be a small Placeholder component, which would accept the content for any placeholder? :)

Reply:
There's no point or benefit of creating any new component for this.